Remotion/Validation/Core/Implementation/InvariantValidationMessage.cs

```diff
@@ -36,7 +36,11 @@ public override string Format (CultureInfo culture, IFormatProvider? formatProvi
       ArgumentUtility.CheckNotNull("culture", culture);
       ArgumentUtility.CheckNotNull("parameters", parameters);
 
-      return string.Format(formatProvider, _validationMessage, parameters);
+
+      using (new CultureScope(CultureInfo.InvariantCulture, culture))
+      {
+        return string.Format(formatProvider, _validationMessage, parameters);
+      }
```

Comment:
```suggestion
      return string.Format(formatProvider ?? CultureInfo.InvariantCulture, _validationMessage, parameters);
```
Given the other comment, I think this code makes it easier to read/align production and test code. Semantically it makes no difference.